Dockerfile

```diff
@@ -0,0 +1,58 @@
+# This file is part of dirtsand.
+#
+# dirtsand is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# dirtsand is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with dirtsand.  If not, see <http://www.gnu.org/licenses/>.
+
+FROM gcc
```

Comment:
This seems like an odd choice...  We don't need anything specific from versions or configurations of GCC, so why not just use a generic Debian or even Alpine base image?

Reply:
I was looking at an example dockerfile when making this choice. Will the debian or alpine images result in smaller containers?

Reply:
Yes, much...  The gcc:latest image is about 400M, whereas debian:stable is ~50M and alpine:latest is ~3M...  The downside though is that you may have to install more packages (e.g. gcc), but somehow I bet the result will still be smaller.